### PR TITLE
ventoy: 1.0.90 -> 1.0.91

### PR DESCRIPTION
**Pull request by AndersonTorres**
Still based on the binary package.

Also closes #223273.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/cd-dvd/ventoy/default.nix [hunk 1]
```diff
@@ -49,15 +49,15 @@ let
   }.${stdenv.hostPlatform.system}
     or (throw "Unsupported platform ${stdenv.hostPlatform.system}");
 in
-stdenv.mkDerivation (finalAttrs: {
```
Comment (SuperSandro2000) [inline pkgs/tools/cd-dvd/ventoy/default.nix:52]: The standard name is finalAttrs and not self

File: pkgs/tools/cd-dvd/ventoy/default.nix [hunk 2]
```diff
@@ -183,7 +184,7 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
```
Comment (SuperSandro2000) [inline pkgs/tools/cd-dvd/ventoy/default.nix:187]: Keeping with lib here is not a problem

**Review by SuperSandro2000 — COMMENTED** [review]
you forgot to carry over the alias entry

File: pkgs/tools/cd-dvd/ventoy/default.nix [hunk 3]
```diff
@@ -200,10 +201,9 @@ stdenv.mkDerivation (finalAttrs: {
...
     '';
     changelog = "https://www.ventoy.net/doc_news.html";
-    license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ AndersonTorres ];
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ AndersonTorres ];
     platforms = [ "x86_64-linux" "i686-linux" "aarch64-linux" "mipsel-linux" ];
-    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
-    mainProgram = "ventoy";
+    sourceProvenance = [ lib.sourceTypes.binaryNativeCode ];
```
Comment (SuperSandro2000) [inline pkgs/tools/cd-dvd/ventoy/default.nix:207]: Removing with here does not really improve readability

**Review by bjornfor — COMMENTED** [review]
Why are there so many unrelated changes in a version bump commit? Please factor them out in a separate commit.

File: pkgs/top-level/aliases.nix [hunk 4]
```diff
@@ -1666,6 +1666,8 @@ mapAliases ({
   vaultwarden-vault = vaultwarden.webvault; # Added 2022-12-13
   varnish71 = throw "varnish71 was removed from nixpkgs, because it was superseded upstream. Please switch to a different release"; # Added 2022-11-08
   vdirsyncerStable  = vdirsyncer; # Added 2020-11-08, see https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168
+  ventoy-bin = throw "ventoy-bin has been renamed to ventoy"; # Added 2023-04-12
+  ventoy-bin-full = throw "ventoy-bin-full has been renamed to ventoy-full"; # Added 2023-04-12
```
Comment (SuperSandro2000) [inline pkgs/top-level/aliases.nix:1670]: This should not be a throw if it can be a simple alias.

**Comment by SuperSandro2000** [comment]
> We need to put an entry on aliases.nix?

Yes, it should be written somewhere in the docs. Also the first and second commit belong together :)

**Pull request merged**